### ISSUE TITLE
Recommencing monthly on-line meeting.

### ISSUE THREAD

**Issue by maktrix** [mozillabd/community#5]
## Goal:

Recommencing monthly on-line meeting.
## Success:

Completing 3 consecutive meeting in next 3 months.
## Related Issues & Links:

[Reference meta issues and past issues] #
## Roles:

Owner: maktrix 
Involved: @anistuhin 
## Required:
- [ ] Discuss and decide if there will be a fixed day of a month, Due: Feb 15th
- [ ] Discuss and decide what to use for monthly meeting, Due: Feb 15th
- [ ] Discuss and decide how to keep meeting logs and meeting minutes: Due: Feb 15th
- [ ] Test run a mock meeting with highest possible number of Mozillians, Due: Feb 20th

**Comment by bellayet** [comment]
We should initiate a Google doc for discussion, take the decisions and documentation.

**Comment by maktrix** [comment]
Nobody is discussing; they might have missed this issue. Shall we call for a meeting?

**Comment by bellayet** [comment]
@maktrix Yes, we should call meetings to resolve the community issues. Please take necessary steps and actions.

**Comment by asif21** [comment]
It will be great if we can sit for a online meeting every month on a fixed date.

**Comment by m4hi2** [comment]
@maktrix bhai, are we planning old school IRC meeting or Skype? 
I think a meeting, not on a fixed date but on a fixed day (like every last Friday), will be helpful.

**Comment by maktrix** [comment]
@asif21 it's not only great, it's effective, productive and innovative and all have been proven in past. This will also keep our pace in acceleration and works as a community town hall.

@m4hi2 we are can not plan anything, unless we have a discussion. Day of a month sounds good, on the contrary each month there is a new date, which may (or may not) confuse some of the community members. Let us be open for options and discuss.

Both of you thanks for the heads up; let's poke others to participate in this discussion.

**Comment by anistuhin** [comment]
Yes, I'm taking this forward and will post a doodle shortly in social channels we have. It's the very first thing we need to come up within this running Miid-term along with the community launch on GitHub.
